src/main/java/appeng/api/config/Settings.java

```diff
@@ -55,6 +55,7 @@ public enum Settings {
     CRAFT_VIA_REDSTONE(EnumSet.of(YesNo.YES, YesNo.NO)),
 
     STORAGE_FILTER(EnumSet.allOf(StorageFilter.class)),
+    TYPE_FILTER(EnumSet.allOf(TypeFilter.class)),
```

Comment:
AE2 calls this `StorageChannel` in other parts of the code, so `STORAGE_CHANNEL` would make more sense

Reply:
I want it to be the same as the higher version.